hvpy/helpers.py

```diff
@@ -87,10 +87,10 @@ def createMovie(
     input_params = locals()
     # These are used later on but we want to avoid passing
     # them into queueMovie.
-    input_params.pop("overwrite")
-    input_params.pop("filename")
-    input_params.pop("hq")
-    input_params.pop("timeout")
+    overwrite = input_params.pop("overwrite")
+    filename = input_params.pop("filename")
+    hq = input_params.pop("hq")
+    timeout = input_params.pop("timeout")
```

Comment:
Not sure what happened here.

Reply:
We just removed this from input_params, I think we don't need to store them.

Reply:
Why don't we need to store them?